doc/modules/model_evaluation.rst

```diff
@@ -467,6 +467,8 @@ given binary ``y_true`` and ``y_pred``:
     * Normalized class-wise accuracy average as described in [Guyon2015]_
     * Macro-average recall as described in [Mosley2013]_
 
+    Note that none of these different definitions are currently implemented in scikit-learn.
```

Comment:
But macro-average recall is implemented? @jnothman

Reply:
I meant not implemented within `balanced_accuracy`